src/jwplayer/players/shared/JwPlayerWrapperWithStrategyRules.tsx

```diff
@@ -39,6 +39,8 @@ const JwPlayerWrapperWithStrategyRules: React.FC<JwPlayerWrapperProps> = ({
 }) => {
 	const { playlistUrl } = config;
 	const strategyRulesPlacementId = 'yXlW7CsI';
+	const playlistId = playlistUrl.match(/playlists\/([a-z0-9]+)/i)?.[1];
+	const playlistKeyVal = playlistId ? `playlist_id=${playlistId}` : `playlist=${playlistUrl}`;
```

Comment:
Consider changing the code earlier: https://github.com/Wikia/jwplayer-fandom/blob/0694c7639f34d557fd0c10d438267b65e8176b86/src/jwplayer/utils/articleVideo/articleVideoConfig.ts#L65-L67 

We have the `playlistId` when creating the article video config and based on that we set up `playlistUrl`. Have you tried removing `playlistUrl` usage in the strategy rules wrapper and just going with the `playlistId`? 🤔

Reply:
will you still be passing us `videoId`s or always a playlist ID? if you will be passing us `videoId`s still then please send it as `media_id=${videoId}`

Reply:
Since last week it should be only a playlist ID :)